lib/public/views/Tags/Details/TagDetailsPage.js

```diff
@@ -11,93 +11,64 @@
  * or submit itself to any jurisdiction.
  */
 
-import { h } from '/js/src/index.js';
+import { h, switchCase } from '/js/src/index.js';
 import errorAlert from '../../../components/common/errorAlert.js';
 import spinner from '../../../components/common/spinner.js';
 import { table } from '../../../components/common/table/table.js';
-import targetURL from '../../../utilities/targetURL.js';
-import { reducedLogsActiveColumns } from '../../Logs/ActiveColumns/reducedLogsActiveColumns.js';
 import { tagDetail } from '../../../components/tag/tagDetail.js';
 import { tabLink } from '../../../components/common/navigation/tabLink.js';
+import { logsActiveColumns } from '../../Logs/ActiveColumns/logsActiveColumns.js';
+import { TAG_DETAILS_PANELS_KEYS } from './TagDetailsModel.js';
 
 /**
  * The VNode of the Tag Detail screen.
  *
- * @param {*} model Pass the model to access the defined functions.
- * @return {vnode} The VNode of the Tag Detail screen.
+ * @param {Model} model Pass the model to access the defined functions.
+ * @return {Component} The VNode of the Tag Detail screen.
```

Comment:
In this scenario I think a `div` will be returned in all cases. I do not think there is a point in specifying `Component`